docs/docs/vite.md

```diff
@@ -359,21 +359,23 @@ There are a few options to resolve this issue:
 - Upgrade the dependency to a version that supports ESM
 - Replace the dependency with an alternative that supports ESM
 
-Failing those solutions, `sku` provides a [`__UNSAFE_EXPERIMENTAL__cjsInteropDependencies`][cjs interop] configuration option can be used as a last resort:
+Failing those solutions, `sku` provides a [`compilePackages`][compilePackages] option that will compile the given modules as if they were part of your source code.
+This may affect build time, but allows vite to interop the CJS code.
```

Comment:
`interop the CJS code` could be reworded. Maybe "allows Vite to handle certain CJS dependencies without throwing the error above".